lib/src/base-styles/_nx-lists.scss

```
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  color: $nx-text-color-light;
+  display: grid;
+  grid-template-columns: 85% auto;
+  grid-template-rows: auto auto;
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
   }
+}
 
-  &.nx-error {
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-  }
+.nx-list__text {
+  @include semi-bold();
 
-  > .fa, > .svg-inline--fa {
-    margin-right: $nx-spacing-md;
-  }
+  grid-column: 1;
+  grid-row: 1;
 }
 
-.nx-list--bulleted {
-  .nx-list__item {
-    @include regular();
+.nx-list__subtext {
+  @include regular();
 
-    border: none;
-    list-style: disc outside;
-    margin: $nx-spacing-sm 10px $nx-spacing-sm 36px;
-    overflow: visible;
-    padding: 0;
-    white-space: normal;
-  }
+  align-self: start;
+  display: block;
+  grid-column: 1;
+  grid-row: 2;
+  // top margin is measured down from the top of the objects grid row
+  margin: $nx-spacing-sm 0 0 0;
```

I believe this is supposed to be 16px
```suggestion
  margin: $nx-spacing-md 0 0 0;
```